firmware_no_aux.cpp

```diff
@@ -21,196 +23,190 @@ Adafruit_NeoPixel pixels(NUMPIXELS, PIN, NEO_GRB + NEO_KHZ800);
+        {'Y', 29},
+        {'Z', 19},
+        {'1', 62},
+        {'2', 60},
+        {'3', 56},
+        {'4', 48},
+        {'5', 32},
+        {'6', 33},
+        {'7', 35},
+        {'8', 39},
+        {'9', 47},
+        {'0', 63}
+};
+
+#define N_MORSE  (sizeof(morseTab)/sizeof(morseTab[0]))
 
 #define SPEED  (12)
 #define DOTLEN  (1200/SPEED)
 #define DASHLEN  (3*(1200/SPEED))
 
-int LEDpin = 12 ;
-
-void
-dash()
-{
-  checkmessage();
-  digitalWrite(LEDpin, HIGH) ;
-  if (light_on == true){
-    beacon_on();
-  }
-  delay(DASHLEN);
-  digitalWrite(LEDpin, LOW) ;
-  if (light_on == true){
-    beacon_off();
-  }
-
-  delay(DOTLEN);
+void dash() {
+    checkMessage();
+    digitalWrite(LED_PIN, HIGH);
+    if (light_on) {
+        beacon_on();
+    }
+    delay(DASHLEN);
+    digitalWrite(LED_PIN, LOW);
+    if (light_on) {
+        beacon_off();
+    }
+    delay(DOTLEN);
 }
 
-void
-dit()
-{
-  checkmessage();
-  digitalWrite(LEDpin, HIGH) ;
-  if (light_on == true){
-     beacon_on();
-   }
-  delay(DOTLEN);
-  digitalWrite(LEDpin, LOW) ;
-  if (light_on == true){
-    beacon_off();
-  }
-  delay(DOTLEN);
+void dit() {
+    checkMessage();
+    digitalWrite(LED_PIN, HIGH);
+    if (light_on) {
+        beacon_on();
+    }
+    delay(DOTLEN);
+    digitalWrite(LED_PIN, LOW);
+    if (light_on) {
+        beacon_off();
+    }
+    delay(DOTLEN);
 }
 
-void
-send(char c)
-{
-  int i ;
-  if (c == ' ') {
-    Serial.print(c) ;
-    delay(7*DOTLEN) ;
-    return ;
-  }
-  for (i=0; i<N_MORSE; i++) {
-    if (morsetab[i].c == c) {
-      unsigned char p = morsetab[i].pat ;
-      Serial.print(morsetab[i].c) ;
-
-      while (p != 1) {
-          if (p & 1)
-            dash() ;
-          else
-            dit() ;
-          p = p / 2 ;
-      }
-      delay(2*DOTLEN) ;
-      return ;
+void send(char c) {
+    int i;
+    if (c == ' ') {
+        Serial.print(c);
+        delay(7 * DOTLEN);
+        return;
+    }
+    for (i = 0; i < N_MORSE; i++) {
+        if (morseTab[i].c == c) {
+            unsigned char p = morseTab[i].pat;
+            Serial.print(morseTab[i].c);
+
+            while (p != 1) {
+                if (p & 1)
+                    dash();
+                else
+                    dit();
+                p = p / 2;
+            }
+            delay(2 * DOTLEN);
+            return;
+        }
     }
-  }
-  /* if we drop off the end, then we send a space */
-  Serial.print("?") ;
+    /* if we drop off the end, then we send a space */
+    Serial.print("?");
 }
 
-void
-sendmsg(char *str)
-{
-  while (*str)
-    send(*str++) ;
-  Serial.println("");
+void sendMsg(char *str) {
+    while (*str)
+        send(*str++);
+    Serial.println("");
 }
 
 
-//Radio shtuff
-RH_ASK driver(2000, 7, 12, 5); //Which will initialise the driver at 2000 bps, recieve on GPIO7, transmit on GPIO12, PTT on GPIO5. This is very poorly documented. PTT is not needed, but whatever. 
+//Radio stuff
+RH_ASK driver(2000, 7, LED_PIN, BUZZER_PIN);
```

Comment:
@TheBoss313 Sorry for my poor documentation. The final argument (PTT) probably should be an unused pin (i.e 4), not the buzzer pin.

Reply:
I'm juggling between a custom modulation (CW, the morse section) and ASK packets via the RadioHead library

Reply:
Probably I'll switch to something else soon. Leave this for now though